package-lock.json

```diff
@@ -0,0 +1,9457 @@
+{
```

Comment:
I think this file should be removed since we're already using yarn, install deps using yarn